common/interfaces/user_interface/ui_delay.h

```diff
@@ -12,6 +12,7 @@
 #define UI_DELAY_H
 
 #include "ui_common.h"
+#include "ui_events_priv.h"
```

Comment:
include `_priv` in only source files. doing this will indirectly expose the `_priv` header to application

Reply:
Fixed